### PR TITLE
[HOTFIX][DOC] Fix the format of SQL in dml-of-carbondata.md

### PR DESCRIPTION
**Pull request by tianyouyangying**
2个语法合并成了一个

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [ ] Any interfaces changed?
 
 - [ ] Any backward compatibility impacted?
 
 - [x] Document update required?

 - [ ] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by zzcclp** [comment]
add to whitelist

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/10872/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/2843/

File: docs/dml-of-carbondata.md [hunk 1]
```diff
@@ -373,9 +373,9 @@ CarbonData DML statements are documented here,which includes:
   Example:
   ```
   LOAD DATA LOCAL INPATH '${env:HOME}/staticinput.csv'
-  INTO TABLE locationTable          
+  INTO TABLE locationTable          ;
```
Comment (qiuchenjian) [inline docs/dml-of-carbondata.md]:
```suggestion
  INTO TABLE locationTable;
```

**Comment by qiuchenjian** [comment]
> 语法有问题

Better to use English to describe your PR，many reviewers and commiters are not chinese

What is the reason for adding ';' ?

**Comment by tianyouyangying** [comment]
> > 语法有问题
> 
> Better to use English to describe your PR，many reviewers and commiters are not chinese
> 
> What is the reason for adding ';' ?



Reason: Load grammar supports two INSERTs and needs to be split into two statements

> > 语法有问题
> 
> Better to use English to describe your PR，many reviewers and commiters are not chinese
> 
> What is the reason for adding ';' ?

Reason: Load grammar supports two INSERTs and needs to be split into two statements

**Comment by zzcclp** [comment]
@tianyouyangying +1 for @qiuchenjian 's suggestion.
1. Please change all chinese to english.
2. Please change PR title to '[HOTFIX][DOC] Fix the format of SQL in dml-of-carbondata.md to avoid ambiguity'
@qiuchenjian The origin SQL in doc actually are two statements, we should seperate them to avoid ambiguity.

Thanks.

File: docs/dml-of-carbondata.md [hunk 2]
```diff
@@ -352,18 +352,18 @@ CarbonData DML statements are documented here,which includes:
   ```
   LOAD DATA [LOCAL] INPATH 'folder_path' 
   INTO TABLE [db_name.]table_name PARTITION (partition_spec) 
-  OPTIONS(property_name=property_value, ...)    
+  OPTIONS(property_name=property_value, ...);
```
Comment (zzcclp) [inline docs/dml-of-carbondata.md]: Better to add a blank line between these two sql.

File: docs/dml-of-carbondata.md [hunk 3]
```diff
@@ -352,18 +352,18 @@ CarbonData DML statements are documented here,which includes:
...
   INSERT INTO INTO TABLE [db_name.]table_name PARTITION (partition_spec) <SELECT STATEMENT>
   ```
 
   Example:
   ```
   LOAD DATA LOCAL INPATH '${env:HOME}/staticinput.csv'
   INTO TABLE locationTable
-  PARTITION (country = 'US', state = 'CA')  
+  PARTITION (country = 'US', state = 'CA') ;
   INSERT INTO TABLE locationTable
```
Comment (zzcclp) [inline docs/dml-of-carbondata.md]: Better to add a blank line between these two sql.

File: docs/dml-of-carbondata.md [hunk 4]
```diff
@@ -373,9 +373,9 @@ CarbonData DML statements are documented here,which includes:
   Example:
   ```
   LOAD DATA LOCAL INPATH '${env:HOME}/staticinput.csv'
-  INTO TABLE locationTable          
+  INTO TABLE locationTable          ;
   INSERT INTO TABLE locationTable
```
Comment (zzcclp) [inline docs/dml-of-carbondata.md]: Better to add a blank line between these two sql.

File: docs/dml-of-carbondata.md (hunk 1 above)
Comment (zzcclp) [reply docs/dml-of-carbondata.md]: +1

File: docs/dml-of-carbondata.md (hunk 2 above)
Comment (kevinjmh) [reply docs/dml-of-carbondata.md]: The seperator is used in beeline , but not in `spark.sql(...)`.
Use blank line to seperate OR use different code block of Markdown instead.

**Comment by QiangCai** [comment]
better to keep the same style to separate SQL.

**Comment by zzcclp** [comment]
@tianyouyangying I have raised another pr(PR#3151) including this modification, please close this pr.

**Comment by tianyouyangying** [comment]
Because Zhichao Zhang had updated DOC and updated this part yesterday, it was shut down directly.

**Pull request closed without merging**